Cargo.lock

```diff
@@ -247,9 +247,9 @@ dependencies = [
 
 [[package]]
 name = "assert_cmd"
-version = "2.0.6"
+version = "2.0.7"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ba45b8163c49ab5f972e59a8a5a03b6d2972619d486e19ec9fe744f7c2753d3c"
+checksum = "fa3d466004a8b4cb1bc34044240a2fd29d17607e2e3bd613eb44fd48e8100da3"
```

Comment:
It looks like you may have run `cargo update` which pulls in many additional silent package version changes. We would like to handle those separate from this PR. Please revert these update changes.

Reply:
That was me, while attempting to fix something a week or two ago. I can undo that.

Reply:
heh. I appear to have made it worse.
Not really sure how to fix that... checking out an old version of Cargo.lock , then running make check, seems to have made it worse.
Similarly, just removing the file and letting it re-generate from make-check, seems to not help.

Any recommendation Bruce?

Reply:
Checking out the old version of `Cargo.lock` and running `cargo check` should put it in the right state. Removing the file would cause it to update all of the packages.

Reply:
hmm that's odd. What I did was checkout the Cargo.lock from the first commit of this branch. Then ran `cargo check`.

I guess that could be flawed if `cargo update` were run as part of the first commit ? That doesn't seem to be the case though, I'm only seeing Cargo.toml modified in the first commit :thinking:

Reply:
I'd recommend checking out the Cargo.lock file from the parent of this branch and then regenerating rather than from the first commit in-case it updated it. Looks like you took care of it already though 😄 Thanks!

Reply:
Ah. I had taken the first commit. But I just repeated the effort with the parent commit and see there are no changes, so I guess that confirms the first commit didn't update it.
:+1: